packages/@uppy/transloadit/src/index.js

```diff
@@ -126,6 +127,10 @@ export default class Transloadit extends BasePlugin {
     addPluginVersion('Zoom', 'uppy-zoom')
     addPluginVersion('Url', 'uppy-url')
 
+    if (this.opts.clientName != null) {
+      list.push(this.opts.clientName)
```

Comment:
Shouldn't we add a prefix for consistency?
```suggestion
      list.push(`client-name:${this.opts.clientName}`)
```

Reply:
I am not sure about this. Currently, the format we use is `$sdk_name:$sdk_version` (or plugin for Uppy). I my mind, the `clientName` is semantically more similar to `$sdk_name` than `$sdk_version`.

That being said, we don't enforce a format right now and people have put other stuff in there as well, so I am unsure how strict we should be here:

![0A8776CF-5F6A-4B15-97EA-D53C2F7B75FE_4_5005_c](https://github.com/transloadit/uppy/assets/1375043/80aaf1a3-1dd4-4298-ac33-bcb280fa18ec)

Reply:
I don't really like the inconsistency, but given the value is empty by default, that's not a big deal. I guess users can still put a colon in the `clientName` themselves.